samples/MeteoControl/app/application.cpp

```diff
@@ -54,7 +54,7 @@ void init()
 	lcd.createChar(6, icon_cross);
 	lcd.createChar(7, icon_check);
 
-	WifiStation.config(ActiveConfig.NetworkSSID, ActiveConfig.NetworkPassword);
+	WifiStation.config(ActiveConfig.NetworkSSID.c_str(), ActiveConfig.NetworkPassword.c_str());
```

Comment:
> As you probably know, there's no way to measure heap fragmentation on ESP. At least not yet.

Not really. Umm-Malloc can give you a nice table with allocated and free memory.  It is possible to disable umm-malloc's allocation functions and just use the stats. It requires work, but it is not impossible.